<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerDialog</class>
 <widget class="QDialog" name="PlayerDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>619</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MJPEG player</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/new/controls/res/video.png</normaloff>:/new/controls/res/video.png</iconset>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="lytMain">
    <item>
     <layout class="QVBoxLayout" name="lytLeft">
      <item>
       <widget class="QFrame" name="frmImage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sldPosition">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
        <property name="pageStep">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="tickPosition">
         <enum>QSlider::NoTicks</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="lytControls">
        <item>
         <widget class="QPushButton" name="pbPrev">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/new/player_controls/res/player_start.png</normaloff>:/new/player_controls/res/player_start.png</iconset>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbBack">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/new/player_controls/res/player_rev.png</normaloff>:/new/player_controls/res/player_rev.png</iconset>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbPlay">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/new/player_controls/res/player_play.png</normaloff>:/new/player_controls/res/player_play.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbForward">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/new/player_controls/res/player_fwd.png</normaloff>:/new/player_controls/res/player_fwd.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbNext">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/new/player_controls/res/player_end.png</normaloff>:/new/player_controls/res/player_end.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbSetStart">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/new/player_controls/res/set_start.png</normaloff>:/new/player_controls/res/set_start.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbSetEnd">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/new/player_controls/res/set_end.png</normaloff>:/new/player_controls/res/set_end.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbSave">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/new/player_controls/res/save.png</normaloff>:/new/player_controls/res/save.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lblStatus">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string>#</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="lytRight">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Source:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbCameras"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbDates"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Records:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="lstRecords"/>
      </item>
      <item>
       <widget class="QPushButton" name="pbClose">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked()</signal>
   <receiver>PlayerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>724</x>
     <y>577</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
